doc/statistics/univariate_statistics.qbk

```diff
@@ -203,7 +203,7 @@ compatible with C++11. In order to pass an ExecutionPolicy to the methods and ac
 Many of these functionals have trivial naive implementations, but experienced programmers will recognize that even trivial algorithms are easy to screw up, and that numerical instabilities often lurk in corner cases.
 We have attempted to do our "due diligence" to root out these problems-scouring the literature for numerically stable algorithms for even the simplest of functionals.
 
-/Nota bene/: Some similar functionality is provided in [@https://www.boost.org/doc/libs/1_68_0/doc/html/accumulators/user_s_guide.html Boost Accumulators Framework].
+/Nota bene/: Some similar functionality is provided in [@https://www.boost.org/doc/libs/1_75_0/doc/html/accumulators/user_s_guide.html Boost Accumulators Framework].
```

Comment:
@jzmaddock : Is there a way to always link to the latest boost version?

Reply:
@NAThompson : use https://www.boost.org/doc/libs/release/doc/html/accumulators.html

Reply:
@mborland : Could you make that change?

Reply:
@NAThompson Done.